src/MapViewController.swift

```diff
@@ -17,7 +17,54 @@ import OnTheRoad
   case generalError, annotationDoesNotExist, apiKeyNotSet, routeDoesNotExist
 }
 
-open class MapViewController: UIViewController, LocationManagerDelegate, TGRecognizerDelegate {
+public protocol MapSingleTapGestureDelegate : class {
```

Comment:
Oh? Interesting. That makes sense given weak referencing the value types would make no sense.